libs/api/domains/national-registry/src/lib/models/user.model.ts

```diff
@@ -1,4 +1,16 @@
-import { Field, ObjectType, ID } from '@nestjs/graphql'
+import { Field, ObjectType, ID, registerEnumType } from '@nestjs/graphql'
+import { IsEnum } from 'class-validator'
+import { Gender } from '../types/gender.enum'
+import { MaritalStatus } from '../types/maritalStatus.enum'
+import { BanMarking } from './banMarking.model'
+
+registerEnumType(MaritalStatus, {
+  name: 'MaritalStatus',
+})
+
+registerEnumType(Gender, {
+  name: 'Gender',
+})
```

Comment:
Not sure if it's correct to register widely used enums in the `user.model` file

Reply:
Yeah agree. Got any suggestion where?

Reply:
Most likely with some interaction with the `index` file... either the enums should be registered there, or they should be defined in `scalars.ts` file (whould choose that option) and with an `export * from './scalars'` in the index file.